pypesto/store/save_to_hdf5.py

```diff
@@ -130,17 +130,20 @@ def write(self, result: Result, overwrite=False):
             for start in result.optimize_result.list:
                 start_id = start['id']
                 start_grp = get_or_create_group(results_grp, start_id)
-                start['history'] = None  # TOOD temporary fix
                 if not overwrite:
                     for key in start.keys():
+                        if key == 'history':
+                            pass
```

Comment:
don't we want `continue` here? Same below.

Reply:
Thanks for the feedback. I'll fix this as soon as possible and update the PR.